bundler/spec/dependabot/bundler/update_checker/version_resolver_spec.rb

```diff
@@ -384,18 +445,29 @@
             to eq(Gem::Version.new("2.0.1"))
         end
 
-        context "that isn't satisfied by the dependencies" do
+        context "that isn't satisfied by the dependencies", :bundler_v1_only do
           let(:dependency_files) do
             project_dependency_files("bundler1/imports_gemspec_version_clash_old_required_ruby_no_lockfile")
           end
-          let(:gemfile_fixture_name) { "imports_gemspec_version_clash" }
           let(:current_version) { "3.0.1" }
 
           it "ignores the minimum ruby version in the gemspec" do
             expect(resolver.latest_resolvable_version_details[:version]).
               to eq(Gem::Version.new("7.2.0"))
           end
         end
+
+        context "that isn't satisfied by the dependencies", :bundler_v2_only do
+          let(:dependency_files) do
+            project_dependency_files("bundler2/imports_gemspec_version_clash_old_required_ruby_no_lockfile")
+          end
+          let(:current_version) { "3.0.1" }
+
+          it "raises a DependencyFileNotResolvable error" do
```

Comment:
Previously it looked like bundler v1 `ignores the minimum ruby version in the gemspec` (see spec above), I think this new behaviour is correct as the currently installed version of `statesman` doesn't satisfy the ruby version in the project gemspec.